src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -36,14 +38,20 @@ public static class LocalizerConstants {
         public static final Vector2 ROBOT_CENTRIC_TURRET_CENTER = new Vector2(3.93, -4);
     }
 
-    @Exclude
+    @Log.Exclude
+    @Config.Exclude
     private final DrivebaseSubsystem drivebaseSubsystem;
-    @Exclude
+
+    @Log.Exclude
+    @Config.Exclude
     private final ShooterSubsystem shooterSubsystem;
-    @Exclude
+
+    @Log.Exclude
+    @Config.Exclude
     private final ShooterVisionSubsystem shooterVisionSubsystem;
 
     private final TimeBasedMedianFilter distanceFilter;
+    private final LinearFilter yawPass;
```

Comment:
Is there a reason to used the time-based filter for distance and a moving average filter for yaw?

Reply:
The distance filter is something we are going to test later in today's meeting